api_tests/preprints/views/test_preprint_detail.py

```diff
@@ -242,6 +243,23 @@ def test_update_primary_file(self, app, user, preprint, url):
         assert log.action == 'preprint_file_updated'
         assert log.params.get('preprint') == preprint._id
 
+    def test_update_preprints_with_no_type(self, app, user, preprint, url):
+        payload = {
```

Comment:
Why are you building a payload by hand rather than using `build_preprint_update_payload()`?

Reply:
Seconded

Reply:
By assigning the content_type to be None by using the build_preprint_update_payload(), it will raise python type check error since content_type expect type 'str' but got 'None' instead. So in this case, build a small dict just for this test is more convenient.